paper/basic_training.tex

```diff
@@ -1211,10 +1197,51 @@ \subsubsection{Grid based Ewald summation}
+This means that if your system is complex and answering your questions will require sampling critical events that have a timescale of seconds or longer, MD will not be the right tool for the job.
+You could invest a huge amount of effort running a MD simulations and find that these do nothing whatsoever to address your questions.
+
+Ideally, you should have some information before beginning that the relevant timescales for your system might be accessible given the amount of MD you can afford to run, or you could plan a set of exploratory MD simulations to assess feasibility. 
+But don't simply plunge in and attempt to run simulations until you find the answers to your questions, as the required timescales could end up being orders of magnitude longer than what you can afford to spend.
+Time is only one consideration out of many; disk storage and computer time availability can also prove limiting factors, and opportunity cost, as well, is not to be overlooked.
+
+Ultimately, we ought to be assessing whether MD is the best tool for the job.
+For our problem of interest, will it really be faster to answer your questions using an MD simulation, or are there experiments which could be done which would answer the question more quickly? 
+Maslow famously wrote, ``I suppose it is tempting, if the only tool you have is a hammer, to treat everything as if it were a nail.''
+We don't want to end up in a position where we are running MD simulations not because they are the best tool for the job, but because they are the only tool available to us.
+If an experiment could answer our key questions with far less cost and time, and the questions are indeed compelling, perhaps our time might be better spent finding a suitable experimental collaborator rather than running a set of simulations.
+To give a concrete example, one could imagine using molecular dynamics simulations to screen a library of commercially available compounds for binding to a potential protein target, but if the compounds are commercially available at an inexpensive rate and a suitable assay is available, it might be far faster and cheaper to simply screen the compounds.
+
+So, count the cost of your potential simulations, assess whether they realistically have a chance of answering the questions you seek to answer, and then carefully decide whether the likelihood of success is worth the cost. 
+If not, don't tackle that problem with MD.
+
+
+\todo[inline, color={yellow!20}]{DLM: Perhaps also a brief ``what NOT to do with your MD data'' blurb, e.g., don't just make movies and look at them. Don't treat them as the answer. Don't overinterpret, etc.}
+
 
+\section{Conclusions}
+Molecular simulations are particularly exciting, because they provide a detailed view into the structure and function of systems at a molecular or atomistic level, and allow us to precisely compute thermodynamic and statistical properties and connect these to the underlying motions, structure, and function.
```

Comment:
Check sentence structure here (making picky notes since we'll only have so much time for revisions at this point).

Reply:
I think it's grammatically correct, but would you prefer two sentences?

Reply:
Right, it's just a little awkward.  And maybe remove "the" before underlying... ?